addons/web/static/src/js/views/control_panel/search/groupby_menu.js

```diff
@@ -78,6 +78,11 @@ var GroupByMenu = DropdownMenu.extend({
     start: function () {
         var superProm = this._super.apply(this, arguments);
         this.$menu.addClass('o_group_by_menu');
+
+        this._computeMaxHeight();
+        var computeMaxHeight = this._computeMaxHeight.bind(this);
+        $(window).on('resize', computeMaxHeight);
```

Comment:
This handler is never unbound. Why not computing height when the user opens the dropdown?

Reply:
I think it should be good this time. The resize event is necessary in the case that the user resizes the window when the dropdown is open.